libs/platform/src/lib/table/table.component.scss

```diff
@@ -232,6 +232,11 @@ $fd-table-header-color-var: var(--sapList_HeaderBackground, #f2f2f2);
                 .fd-table__popover--custom {
                     display: none;
                 }
+
```

Comment:
I am fine as a workaround until this is brought to fundamental-styles. After this is merged, can you maintain it in styles, bring the change from there here and remove this styling?

Reply:
Sure.